examples/listen-psk/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/pion/dtls"
+	"github.com/pion/dtls/examples/util"
+)
+
+func main() {
+	// Prepare the IP to connect to
+	addr := &net.UDPAddr{IP: net.ParseIP("127.0.0.1"), Port: 4444}
+
+	//
+	// Everything below is the pion-DTLS API! Thanks for using it ❤️.
+	//
+
+	// Prepare the configuration of the DTLS connection
+	config := &dtls.Config{
+		PSK: func(hint []byte) ([]byte, error) {
```

Comment:
Same thing as in dial-psk, use the blank identifier instead of `hint`?

Reply:
Addressed above https://github.com/pion/dtls/pull/76/commits/0a9c0ced8977b98ce9037943e11b4d089f696ca0